docs/data/run-frontier-squid-container.md

```diff
@@ -0,0 +1,136 @@
+Running Frontier Squid in a Container
+=====================================
+
+Frontier Squid is a distribution of the well-known [squid HTTP caching
+proxy software](http://squid-cache.org) that is optimized for use with
+applications on the Worldwide LHC Computing Grid (WLCG). It has
+[many advantages](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Why_use_frontier_squid_instead_o)
+over regular squid for common grid applications, especially Frontier
+and CVMFS. The OSG distribution of frontier-squid is a straight rebuild of the
+upstream frontier-squid package for the convenience of OSG users.
+
+!!! tip
+    OSG recommends that all sites run a caching proxy for HTTP and HTTPS to help reduce bandwidth and improve
+    throughput.
+
+This document outlines how to run Frontier Squid in a Docker container.
+
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following points (consulting [the Frontier Squid Reference section](/data/frontier-squid/#reference) as needed)::
+
+1. **Docker:** For the purpose of this guide, the host must have a running docker service
+   and you must have the ability to start containers (i.e., belong to the `docker` Unix group).
+1.   **Network ports:** Frontier squid communicates on ports 3128 (TCP) and 3401 (UDP).
+     - We encourage sites to allow monitoring on port 3401 via UDP from CERN IP address ranges, 128.142.0.0/16,
+       188.184.128.0/17, 188.185.48.0/20 and 188.185.128.0/17.
+     - If utgoing connections are filtered, note that CVMFS always uses ports 8000, 80, or 8080.
```

Comment:
```suggestion
     - We encourage sites to allow monitoring on port 3401 via UDP from CERN IP address ranges, 128.142.0.0/16,
       188.184.128.0/17, 188.185.48.0/20 and 188.185.128.0/17.
     - If outgoing connections are filtered, note that CVMFS always uses ports 8000, 80, or 8080.
```

In addition to the typo fix above, let's link to the CERN doc for IP ranges instead of listing the ranges. That way we don't have to track any changes to the IP ranges ourselves.